compass/ocean/tests/hurricane/forward/forward.py

```diff
@@ -38,15 +45,26 @@ def __init__(self, test_case, mesh, init, name='forward', subdir=None):
         """
         self.mesh = mesh
         self.init = init
+        self.use_lts = use_lts
+
         super().__init__(test_case=test_case, name=name)
 
-        self.add_namelist_file(
-            'compass.ocean.tests.hurricane.forward', 'namelist.ocean')
-        self.add_streams_file(
-            'compass.ocean.tests.hurricane.forward', 'streams.ocean')
+        if use_lts is False:
```

Comment:
This is the preferred way:
```suggestion
        if not use_lts:
```
Though I nearly always do the `if <blah>: ... else:` version rather than the `if not <blah>: ... else:` because in this latter case the `else` has a weird double-negative meaning.

Reply:
I changed it to `if use_lts:` `else:`